src/cart/crt_iv.c

```diff
@@ -2887,16 +2886,16 @@ bulk_update_transfer_done_aux(const struct crt_bulk_cb_info *info)
 			   cb_info->buc_user_priv);
 
 	rc = crt_bulk_free(cb_info->buc_bulk_hdl);
+	output->rc = rc;
+
+	crt_reply_send(info->bci_bulk_desc->bd_rpc);
+	RPC_PUB_DECREF(info->bci_bulk_desc->bd_rpc);
```

Comment:
am not quite sure what this is supposed to accomplish; this looks like no different than it was before, but not an issue to be moved here

Reply:
I would just do cleanup after send reply to avoid any clearing resources before they will be sent.